rmm/monitor/src/rmm/granule.rs

```diff
@@ -2,14 +2,17 @@ use crate::rmm::PageMap;
 extern crate alloc;
 use alloc::collections::btree_map::BTreeMap;
 
-pub const FVP_DRAM0_BASE: usize = 0x8000_0000;
-pub const FVP_DRAM0_SIZE: usize = 0x7C00_0000;
-pub const FVP_DRAM0_END: usize = FVP_DRAM0_BASE + FVP_DRAM0_SIZE - 1;
+use spinning_top::{Spinlock, SpinlockGuard};
+
+const FVP_DRAM0_BASE: usize = 0x8000_0000;
+const FVP_DRAM0_SIZE: usize = 0x7C00_0000;
+const FVP_DRAM0_END: usize = FVP_DRAM0_BASE + FVP_DRAM0_SIZE - 1;
```

Comment:
Since `Range` is half open, we don't need to define *_END.
How about to fix like this? I think we could fix `validate_addr()` more concise too.

```suggestion
const FVP_DRAM0_REGION: core::ops::Range = core::ops::Range { start: 0x8000_0000, end: 0x7C00_0000 });
```

Reply:
applied it